packages/pcd/pod-ticket-pcd/src/PODTicketPCDPackage.ts

```diff
@@ -0,0 +1,244 @@
+      argumentType: ArgumentTypeName.String
+    },
+    entries: {
+      value: ticketDataToEntries(ticket),
+      argumentType: ArgumentTypeName.Object
+    }
+  });
+
+  const id = args.id.value ?? uuid();
+
+  return new PODTicketPCD(id, { ticket }, { podPCD });
+}
+
+/**
+ * Verifies a POD Ticket PCD by checking that its {@link PODTicketPCDClaim} corresponds to
+ * its {@link PODTicketPCDProof}. If they match, the function returns true, otherwise false.
+ * In most cases, verifying the validity of the PCD with this function is not enough.
+ * It may also be necessary to ensure that the parameters of the ticket, such as the
+ * productId and eventId, match the expected values, and that the public key of the
+ * entity that signed the ticket is indeed the authority for that event.
+ */
+export async function verify(pcd: PODTicketPCD): Promise<boolean> {
+  const content = PODContent.fromEntries(ticketDataToEntries(pcd.claim.ticket));
+
+  if (content.contentID !== pcd.proof.podPCD.pod.contentID) {
+    return false;
+  }
+
+  return PODPCDPackage.verify(pcd.proof.podPCD);
+}
+
+/**
+ * Serializes a {@link PODTicketPCD}.
+ * @param pcd The POD Ticket PCD to be serialized.
+ * @returns The serialized version of the POD Ticket PCD.
+ */
+export async function serialize(
+  pcd: PODTicketPCD
+): Promise<SerializedPCD<PODTicketPCD>> {
+  const serializedPODPCD = await PODPCDPackage.serialize(pcd.proof.podPCD);
+
+  return {
+    type: PODTicketPCDTypeName,
+    pcd: JSONBig().stringify({
+      id: pcd.id,
+      podPCD: serializedPODPCD,
+      ticket: pcd.claim.ticket
+    })
+  } as SerializedPCD<PODTicketPCD>;
+}
+
+/**
+ * Deserializes a serialized {@link PODTicketPCD}.
+ * @param serialized The serialized PCD to deserialize.
+ * @returns The deserialized version of the POD Ticket PCD.
+ */
+export async function deserialize(serialized: string): Promise<PODTicketPCD> {
+  const deserializedWrapper = JSONBig().parse(serialized);
+  const deserializedPODPCD = await PODPCDPackage.deserialize(
+    deserializedWrapper.podPCD.pcd
+  );
+  return new PODTicketPCD(
+    deserializedWrapper.id,
+    { ticket: deserializedWrapper.ticket },
+    { podPCD: deserializedPODPCD }
+  );
+}
+
+export function ticketDisplayName(
+  eventName?: string,
+  ticketName?: string
+): string {
+  let displayName = "";
+
+  if (eventName && eventName?.length > 0) {
+    displayName += eventName;
+  }
+
+  if (ticketName && ticketName?.length > 0) {
+    if (displayName.length === 0) {
+      displayName = ticketName;
+    } else {
+      displayName += ` (${ticketName})`;
+    }
+  }
+
+  return displayName.length === 0 ? "untitled" : displayName;
+}
+/**
+ * Provides the information about the {@link PODTicketPCD} that will be displayed
+ * to users on Zupass.
+ * @param pcd The POD Ticket PCD instance.
+ * @returns The information to be displayed, specifically `header` and `displayName`.
+ */
+export function getDisplayOptions(pcd: PODTicketPCD): DisplayOptions {
```

Comment:
This seems like another candidate for splitting out into shared code, since it should probably be the same as what's used for EdDSATickets.

Alternately or additionally: Perhaps some of the default display strings should actually be different so that we can tell a POD ticket from an EdDSA ticket?